orion/database/migrations/000004_create_table_semesters.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE semesters
+(
+    id          int unsigned NOT NULL AUTO_INCREMENT,
+    created_at  bigint(20)   NOT NULL,
+    updated_at  bigint(20)   NOT NULL,
+    deleted_at  datetime,
+    semester_id text         NOT NULL UNIQUE,
```

Comment:
semester_ids are short, so instead of text, make it a varchar(64) as well, I think.